src/app/shared/components/create-project/create-project.component.ts

```diff
@@ -90,11 +90,48 @@ export class CreateProjectComponent implements OnInit, OnDestroy {
         this.form.controls[i].updateValueAndValidity();
       }
     }
-    console.log(this.form.value);
+
     if (this.form.valid && isPrimarySkillsValid) {
-      this.message.success('Project created successfully');
+      const { dates, registrationDates, ...restForm } = this.form.value;
+      const toSend = {
```

Comment:
please move this to the Facade. The component does not need to know about the data structure for the service

Reply:
done.